### PR TITLE
ENH: Functional workflow

### PR DESCRIPTION
**Pull request by mgxd**
Following #7, this PR focuses on compatibility with the BOLD workflows.

### CODE REVIEW

File: fmriprep_rodents/patch/interfaces/__init__.py [hunk 1]
```diff
@@ -26,3 +39,278 @@ def _run_interface(self, runtime):
...
                 )
         return runtime
+
+
+class _TemplateFlowSelectOutputSpec(TFSelectOutputSpec):
+    brain_mask = File(desc="Template's brain mask")
+    t2w_file = File(desc="Template's T2w image")
+
+
+class TemplateFlowSelect(_TFSelect):
```
Comment (oesteban) [inline fmriprep_rodents/patch/interfaces/__init__.py:49]: Should we move the nipype interface to the templateflow Python client codebase?

**Review by oesteban — APPROVED** [review]
I'd keep moving with this, we'll figure the normalization out.

File: fmriprep_rodents/patch/interfaces/__init__.py (hunk 1 above)
Comment (mgxd) [reply fmriprep_rodents/patch/interfaces/__init__.py:49]: Doing so would make the python client fairly heavy (we'd need to add nipype as a dependency), so I'm convinced there.

Ultimately I think we'll need to revise the interface to mirror `templateflow.api.get`.

**Comment by mgxd** [comment]
I'm going to merge this in now so we aren't stuck in limbo, despite it not being fully functional yet.

**Pull request merged**